File: app/src/main/java/fr/free/nrw/commons/campaigns/CampaignConfig.java
package fr.free.nrw.commons.campaigns;

import com.google.gson.annotations.SerializedName;

/**
 * A data class to hold the campaign configs
 */
class CampaignConfig {

    @SerializedName("showOnlyLiveCampaigns") private boolean showOnlyLiveCampaigns;
    @SerializedName("sortBy") private String sortBy;
}

File: app/src/main/java/fr/free/nrw/commons/campaigns/CampaignConfig.kt
package fr.free.nrw.commons.campaigns

import com.google.gson.annotations.SerializedName

/**
 * A data class to hold the campaign configs
 */
class CampaignConfig {
    @SerializedName("showOnlyLiveCampaigns")
    var showOnlyLiveCampaigns = false

    @SerializedName("sortBy")
    var sortBy: String? = null
}